### DISCUSSION TITLE
Could the license for this repo be updated to an "OSI-approved" license?

### DISCUSSION THREAD

**Discussion by pmoprhy** [krzyzanowskim/CryptoSwift#1052] [General]
In order to use this library I'd have to jump through a lot of hoops to get it approved with my company because the license doesn't appear to be approved by the Open Source Initiative (OSI).  For example:
MIT License
Apache License 2.0
GNU General Public License (GPL) v2.0 and v3.0
BSD 2-Clause "Simplified" License
BSD 3-Clause "New" or "Revised" License
Mozilla Public License 2.0
Eclipse Public License 2.0
GNU Lesser General Public License (LGPL) v2.1 and v3.0
@krzyzanowskim, would you be open to changing the license ?

**Comment by krzyzanowskim** [comment]
no, thank you, I'm good.
It's too much work, and I don't want to.

**Reply by pmoprhy** [reply]
ok, thanks for the quick response
